src/repository/comments.py

```diff
@@ -37,3 +56,66 @@ async def create_comment (
     db.refresh(comment)
     return comment
     
+
+async def get_comments(
+        photo_id: int, 
+        limit: int, 
+        offset: int,
+        db: Session,
+        ) -> List[PhotoComment]:
+    """
+    The get_comments function returns a list of comments for the photo with the given id.
+        The limit and offset parameters are used to paginate through results.
+        
+    :param photo_id: int: Filter the comments by photo id
+    :type photo_id: int
+    :param limit: int: Limit the number of comments returned
+    :type limit: int
+    :param offset: int: Specify the number of records to skip before starting to return rows
+    :type offset: int
+    :param db: Session: Pass the database session to the function
+    :type db: Session
+    :return: A list of photocomment objects
+    :rtype: List[PhotoComment] 
+    """
+    stmt = Select(PhotoComment).filter_by(photo_id=photo_id).order_by(asc(PhotoComment.id)).offset(offset).limit(limit)
+    contacts =  db.execute(stmt)
+    return contacts.scalars().all()
+
+
+async def update_comment(
+        comment_id: int, 
+        photo_id: int,      
+        updated_comment: CommentSchema, 
+        current_user: User,
+        db: Session,
+        ):
+    """
+    The update_comment function updates a comment in the database.
+    
+    :param comment_id: int: Identify the comment to be updated
+    :type comment_id: int
+    :param photo_id: int: Find the comment that is associated with the photo
+    :type photo_id: int
+    :param updated_comment: CommentSchema: Pass the updated comment to the function
+    :type updated_comment: CommentSchema
+    :param current_user: User: Check if the user is logged in and has permissions to update a comment
+    :type current_user: User
+    :param db: Session: Pass in the database session
+    :param : Get the id of the comment to be deleted
+    :return: A comment object
+    :doc-author: Trelent
+    """
+    stmt = Select(PhotoComment).filter_by(id=comment_id, photo_id=photo_id, created_by=current_user.id)
```

Comment:
Rename 'stmt' to something understandable.

Reply:
done